src/utils/stringFormat.tsx

```diff
@@ -110,3 +110,11 @@ export function combineScientificName(entries: Entry[]): Entry[] {
 
   return newEntries;
 }
+
+/**
+ * Returns the error message of the given error.
+ * @param err the error
+ */
+export function getErrorMessage(err: any): string {
```

Comment:
```suggestion
export function getErrorMessage(err: unknown): string {
```

Reply:
Can't access `err.response.data` if `err` is `unknown`. Could make an interface for errors instead? (would prefer to do this in a separate ticket)